pkg/clients/generated/apis/run/v1beta1/runservice_types.go

```diff
@@ -388,7 +388,7 @@ type ServiceTraffic struct {
 type ServiceValueSource struct {
 	/* Selects a secret and a specific version from Cloud Secret Manager. */
 	// +optional
-	SecretKeyRef *v1alpha1.ResourceRef `json:"secretKeyRef,omitempty"`
+	SecretKeyRef *v1alpha1.SecretKeyRef `json:"secretKeyRef,omitempty"`
```

Comment:
This is actually neither a ResourceRef, nor a SecretKeyRef.

It has two properties which are both of type ResourceRef.

Reply:
I think we can address this issue separately in future, as long as we don't assume all fields named secretKeyRef are of type "SecretKeyRef".